code/_onclick/item_attack.dm

```diff
@@ -133,7 +133,7 @@
 // Click parameters is the params string from byond Click() code, see that documentation.
 /obj/item/proc/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
 	SEND_SIGNAL(src, COMSIG_ITEM_AFTERATTACK, target, user, proximity_flag, click_parameters)
-	SEND_SIGNAL(user, COMSIG_MOB_ITEM_AFTERATTACK, target, user, proximity_flag, click_parameters)
+	SEND_SIGNAL(user, COMSIG_MOB_ITEM_AFTERATTACK, target, src, proximity_flag, click_parameters)
```

Comment:
is this fixing a bug? if so this is pretty major

Reply:
![image](https://user-images.githubusercontent.com/110184118/223569254-1cb984e4-e5e1-4146-9937-7ce6faa2a3ca.png)
The original BIDDLE heretic PR from TG includes this, can't find anything about it in the changelog.

Reply:
Then we should investigate what argument is changing here and why

Reply:
![image](https://user-images.githubusercontent.com/10366817/225749157-be8ec096-9f31-4ded-98d6-ac537bde6afe.png)